tutorials/advanced/transforms_tutorial.py

```diff
@@ -14,26 +14,33 @@
 generation, it helps to have an understanding of these objects so you
 can reuse the existing transformations matplotlib makes available to
 you, or create your own (see :mod:`matplotlib.transforms`).  The table
-below summarizes the existing coordinate systems, the transformation
+below summarizes the some useful coordinate systems, the transformation
 object you should use to work in that coordinate system, and the
 description of that system. In the `Transformation Object` column,
 ``ax`` is a :class:`~matplotlib.axes.Axes` instance, and ``fig`` is a
 :class:`~matplotlib.figure.Figure` instance.
 
-==========  =====================  ====================================================================================
-Coordinate  Transformation Object  Description
-==========  =====================  ====================================================================================
-`data`      ``ax.transData``       The userland data coordinate system, controlled by the xlim and ylim
-`axes`      ``ax.transAxes``       The coordinate system of the :class:`~matplotlib.axes.Axes`; (0, 0) is
-                                   bottom left of the axes, and (1, 1) is top right of the axes.
-`figure`    ``fig.transFigure``    The coordinate system of the :class:`~matplotlib.figure.Figure`; (0, 0)
-                                   is bottom left of the figure, and (1, 1) is top right of the figure.
-`display`   `None`                 This is the pixel coordinate system of the display; (0, 0) is the bottom
-                                   left of the display, and (width, height) is the top right of the display in pixels.
-                                   Alternatively, the identity transform
-                                   (:class:`matplotlib.transforms.IdentityTransform()`) may be used instead of None.
-==========  =====================  ====================================================================================
-
+=========== ============================= =====================================
+Coordinates Transformation object         Description
+=========== ============================= =====================================
+"data"      ``ax.transData````            The userland data coordinate system,
```

Comment:
What is a "userland"?

Reply:
"from the POV of the user" but I agree it's a bit obscure.  "The data coordinate system" actually sounds enough?

Reply:
force pushed (also fixed below)